pkg/config/run.go

```diff
@@ -68,13 +68,14 @@ type Step struct {
 }
 
 type StepCondition struct {
-	Config                    map[string][]interface{} `json:"config,omitempty"`
-	ConfigKey                 string                   `json:"configKey,omitempty"`
-	FilePattern               string                   `json:"filePattern,omitempty"`
-	FilePatternFromConfig     string                   `json:"filePatternFromConfig,omitempty"`
-	Inactive                  bool                     `json:"inactive,omitempty"`
-	NpmScript                 string                   `json:"npmScript,omitempty"`
-	CommonPipelineEnvironment map[string]interface{}   `json:"commonPipelineEnvironment,omitempty"`
+	Config                                  map[string][]interface{} `json:"config,omitempty"`
+	ConfigKey                               string                   `json:"configKey,omitempty"`
+	FilePattern                             string                   `json:"filePattern,omitempty"`
+	FilePatternFromConfig                   string                   `json:"filePatternFromConfig,omitempty"`
+	Inactive                                bool                     `json:"inactive,omitempty"`
+	NpmScript                               string                   `json:"npmScript,omitempty"`
+	CommonPipelineEnvironment               map[string]interface{}   `json:"commonPipelineEnvironment,omitempty"`
+	CommonPipelineEnvironmentVariableExists []string                 `json:"commonPipelineEnvironmentVariableExists,omitempty"`
```

Comment:
perhaps we should keep it a single string value and then if multiple cpe need to be checked we add the condition multiple times .

Reply:
Can it be implemented as default value i.e. `*` or `<any>` for `CommonPipelineEnvironment`?

Reply:
@anilkeshav27 I made the suggested change and renamed the condition as well.
@raman-susla-epam Thank you for the suggestion. I had considered it as well, but in discussion with Anil we decided to keep it in this way because it's more readable in the stage conditions YAML.